src/components/ToggleSwitch/index.tsx

```diff
@@ -0,0 +1,206 @@
+import * as React from 'react';
+import {
+  StyleSheet,
+  Animated,
+  TouchableOpacity,
+  Easing,
+  StyleProp,
+  Text,
+  TextStyle,
+  Platform,
+  TouchableOpacityProps
+} from 'react-native';
+import dimen from '../../utils/dimen';
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    alignItems: 'center'
+  },
+  basicStyle: {
+    height: dimen.normalizeDimen(16),
+    width: dimen.normalizeDimen(16),
+    borderRadius: dimen.normalizeDimen(16 / 2),
+    backgroundColor: '#2C67BC'
+  },
+  eahcStyles: {
+    fontSize: dimen.normalizeDimen(12),
+    color: '#f4f3f4',
+    position: 'absolute',
+    top: dimen.normalizeDimen(2),
+    left: dimen.normalizeDimen(3)
+  },
+
+  eahcStylesOf: {
+    fontSize: dimen.normalizeDimen(12),
+    color: '#f4f3f4',
+    position: 'absolute',
+    top: dimen.normalizeDimen(2),
+    right: dimen.normalizeDimen(3)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The class name `eahcStylesOf` seems to be a typo. It's better to use meaningful names for classes. This will improve readability and maintainability of the code.
```diff
-   eahcStylesOf: {
+   eachStylesOff: {
    fontSize: dimen.normalizeDimen(12),
    color: '#f4f3f4',
    position: 'absolute',
    top: dimen.normalizeDimen(2),
    right: dimen.normalizeDimen(3)
  },
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->